scripts/agent.yml

```diff
@@ -101,6 +103,9 @@ tasks:
             # file to s3.put
             mkdir upload
             echo ${task_name} > upload/s3
+
+            # miscellaneous files written by commands
+            mkdir output
```

Comment:
What does this do?

Reply:
expansions.write doesn't make parent directories, so I just created it.